pkg/kube/inject/inject.go

```diff
@@ -857,6 +861,12 @@ func IntoObject(sidecarTemplate string, valuesConfig string, meshconfig *meshcon
 	}
 
 	metadata.Annotations[annotation.SidecarStatus.Name] = status
+	if status != "" && metadata.Labels[MTLSReadyLabelName] == "" {
```

Comment:
Do we have a test for the case label is already set?

Reply:
+1

Reply:
added